README.md

```diff
@@ -182,6 +182,11 @@ try {
 We're happy to incorporate all improvements to JPlag into this codebase. Feel free to fork the project and send pull requests.
 Please consider our [guidelines for contributions](https://github.com/jplag/JPlag/wiki/3.-Contributing-to-JPlag).
 
+### Git hooks
```

Comment:
@tsaglam has to say whether https://github.com/jplag/JPlag/wiki/3.-Contributing-to-JPlag is a better place for the information :)

Reply:
We discussed this in the DevMeeting, we concluded the wiki is the right place.